include/mbedtls/ssl.h

```diff
@@ -3751,7 +3751,8 @@ void mbedtls_ssl_conf_groups(mbedtls_ssl_config *conf,
  *                 used for certificate signature are controlled by the
  *                 verification profile, see \c mbedtls_ssl_conf_cert_profile().
  *
- * \deprecated     Superseded by mbedtls_ssl_conf_sig_algs().
+ * \deprecated     Superseded by `mbedtls_ssl_conf_sig_algs()`. See
```

Comment:
Two things: doxygen does not use markdown, so we shouldn't use backquotes here. Also, I don't see how "See also" adds anything to the first sentence, IMO this comment was fine as it was before this commit.

Reply:
Ah, my mistake. I've learned something today, then.